packages/manager/modules/vps/src/dashboard/migrate/index.scss

```diff
@@ -0,0 +1,6 @@
+oui-select-picker {
+  .oui-select-picker__body ul li {
+    justify-content: space-between;
+    display: flex;
```

Comment:
This looks achievable with some bootstrap4 utilities

Reply:
Yes but that means i have to add a class to every `li`, so using the css selector instead